src/main/java/com/ldtteam/structurize/client/BlueprintRenderer.java

```diff
@@ -690,13 +689,11 @@ public static void apply(final float overrideValue)
                 return;
             }
 
-            float alpha = Structurize.getConfig().getClient().rendererTransparency.get().floatValue();
```

Comment:
This won't work, this is dynamic value and needs to be reliazed in this class. Please revert (it might be possible to move this out, but it would rather large refactor to keep the current logic)
Also for this reason there is the `getOverridePreviewTransparency` so if you need some more logic please enhance that
And if you plan to use this in some real scenarios, then it was not built for that, this transparency hack still might result in potential x-rays etc. If we want real transparency for previews then we need much more work

Reply:
I can add the config value back again, but the override will need to take precedence, else the override does not work unless the user manually enabled transparency in the config before. I dont care about artifacts/sometimes broken rendering

Reply:
Yeah, but I care, it's called hack for a reason, and if you want to use this as default for your feature then consider using proper transparency specifically for that
If you really want to use this for w/e, then displaying the same huge warning message (as we do in build tool) is fine with me
_Else you can straight flag this mod with "hack to see through all transparency blocks - use it via preview"_...

Reply:
tbh, it has been working fairly well so far, so I wouldn't be worrying too much about it. Not like it's a production competitive IEEE title. I don't even think proper transparency works in minecraft =D Not aware of anybody that does transparency things that achieved it =D

Reply:
IEEE, lol, I mean AAA =D either way clamp is missing then we can do shipit.